app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,52 @@
+class QuestionsController < ApplicationController
+  before_action :set_question, only: [:show, :update, :destroy]
+
+  def index
+    questions = Question.all
+    render json: questions, status: 200
+  end
+
+  def show
+    render json: @question, status: 200
+  end
+
+  def create
+    question = Question.new(questions_params)
+    if question.save
+      render json: question, status: :created
+    else
+      invalid_request("field can't be blank")
+    end
+  end
+
+  def update
+    if @question.try(:update, questions_params)
+      render json: @question, status: 200
+    else
+       invalid_request("field can't be blank")
```

Comment:
`invalid_request("field can't be blank")` how about when the update couldn't be done due to other errors. `Field can't be blank is a bit generic`